### PR TITLE
UI/mod tools and banned users

### PR DESCRIPTION
**Pull request by tusharravindran**
UI/mod tools and banned Users list for that Community 
My Description Profile changes

### CODE REVIEW

File: app/views/communities/_bannedusers.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="card" style="width: 40rem;">
+  <div class="card-header">Banned Users <i class="fa fa-info-circle text-primary ml-1"></i></div>
+  <% if @banneduser.empty? %>
+    <p>***No Banned Users***</p>
+  <% else %>
+    <% @banneduser.each do |user| %>
```
Comment (iMagesh) [inline app/views/communities/_bannedusers.html.erb:6]: can you change the variable to follow conventions? it has to be snake case when there are more than 1 words

File: app/views/communities/mod.html.erb [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-              <li class="list-group-item"> <%= user.username %> </li>
-            </ul>
-          <% end %>
-        </div>
-      </div>
+<div class="row ">
+  <div class="col-3">
+    <div class="list-group ml-4" id="list-tab" role="tablist">
+      <li class="list-group-item text-white bg-secondary "><i class="fa fa-cog mr-2"></i>User Management</li>
+      <%= link_to mod_path(mod: "Banusers")  do %>
```
Comment (iMagesh) [inline app/views/communities/mod.html.erb:5]: use snake case

File: app/views/public/profile.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <%= render partial: "public/profiletab"%>
```
Comment (iMagesh) [inline app/views/public/profile.html.erb:1]: use snake case for file names as well

File: app/views/public/profile.html.erb [hunk 4]
```diff
@@ -13,7 +13,7 @@
         <%= render partial: "public/mysavedpost"%>
```
Comment (iMagesh) [inline app/views/public/profile.html.erb:13]: use snake case for file names when there are more than one word

File: app/views/public/profile.html.erb [hunk 5]
```diff
@@ -13,7 +13,7 @@
...
       </div>
     </div>
-    <div class="col-4 p-2 ml-5" >
+    <div class="col-sm-4" >
       <%= render partial: "public/mydescription"%>
```
Comment (iMagesh) [inline app/views/public/profile.html.erb:17]: use snake case for file names when there are more than one word

**Review by iMagesh — CHANGES_REQUESTED** [review]
check comments and update the code

File: app/views/communities/_bannedusers.html.erb (hunk 1 above)
Comment (tusharravindran) [reply app/views/communities/_bannedusers.html.erb:6]: Resolved by creating new Branch #113

File: app/views/communities/mod.html.erb (hunk 2 above)
Comment (tusharravindran) [reply app/views/communities/mod.html.erb:5]: Resolved by creating new Branch #113

File: app/views/public/profile.html.erb (hunk 3 above)
Comment (tusharravindran) [reply app/views/public/profile.html.erb:1]: Resolved by creating new Branch #113

File: app/views/public/profile.html.erb (hunk 4 above)
Comment (tusharravindran) [reply app/views/public/profile.html.erb:13]: Resolved by creating new Branch #113

File: app/views/public/profile.html.erb (hunk 5 above)
Comment (tusharravindran) [reply app/views/public/profile.html.erb:17]: Resolved by creating new Branch #113

**Pull request closed without merging**